insights/specs/__init__.py

```diff
@@ -86,7 +86,6 @@ class Specs(SpecSet):
     dumpdev = RegistryPoint()
     dumpe2fs_h = RegistryPoint(multi_output=True)
     engine_config_all = RegistryPoint()
-    engine_log = RegistryPoint()
```

Comment:
This renaming is also not suggested since it would break some existing tests in some unknown project that depends on this spec.
But in fact, it doesn't impact the rules execution in the production environment, it's acceptable.

Reply:
Thanks.